Tests/Admin/AdminTest.php

```diff
@@ -1523,19 +1524,11 @@ public function testGetSubject()
         $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
         $admin->setModelManager($modelManager);
 
-        $admin->setRequest(new Request(array('id' => 'azerty')));
```

Comment:
For now, the result of `$this->subject = $this->getModelManager()->find($this->class, $id);`. Certainly `null`.

For later, integration test with db service for Travis would be awesome. :+1: